### PR TITLE
New Home UI

### PR DESCRIPTION
**Pull request by Warhedgehog**
This PR changes the Home screens UI. It does not touch the scanning part though. Here are screenshots of UIs

### CODE REVIEW

**Review by jbsv — APPROVED** [review]
LGTM. Nice job. some minor comments.

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java [hunk 1]
```diff
@@ -171,31 +177,25 @@ public void savePersistentData() throws GeneralSecurityException {
...
-        .getMessageSender()
-        .publish(keyManager.getMainKeyPair(), Channel.ROOT, createLao)
-        .doOnComplete(() -> Log.d(TAG, "got success result for create lao with id " + lao.getId()))
-        .andThen(networkManager.getMessageSender().subscribe(lao.getChannel()))
-        .toSingleDefault(lao.getId());
+    networkManager.connect(serverAddress);
+    getApplication()
+        .startActivity(
+            ConnectingActivity.newIntentForCreatingDetail(
+                getApplication().getApplicationContext(), laoName));
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java:190]: Ideally, I would prefer to keep the structure where the view model does not change the fragment.

To me, it is the caller's job to do that.

I also join Jean on the Javadoc/method name that doesn't seem to be up-to-date with the actual behavior.

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/wallet/SeedWalletFragment.java [hunk 2]
```diff
@@ -29,8 +32,8 @@
 public class SeedWalletFragment extends Fragment {
 
   public static final String TAG = SeedWalletFragment.class.getSimpleName();
-  private WalletSeedFragmentBinding mWalletSeedFragBinding;
-  private HomeViewModel mHomeViewModel;
+  private WalletSeedFragmentBinding binding;
+  private HomeViewModel viewModel;
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/wallet/SeedWalletFragment.java:36]: 👍

**Review by GabrielFleischer — COMMENTED** [review]
Looks good to me, just a slight proposition in the hom view model

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java (hunk 1 above)
Comment (Warhedgehog) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java]: Agreed, especially considering that the whole thing of going through the viewmodel was not relevant anymore imo

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java [hunk 3]
```diff
@@ -171,31 +177,25 @@ public void savePersistentData() throws GeneralSecurityException {
   }
 
   /**
-   * launchLao is invoked when the user tries to create a new LAO. The method creates a `CreateLAO`
+   * createLao is invoked when the user tries to create a new LAO. The method creates a `CreateLAO`
```
[the remark this answers is not in the document]
Comment (Warhedgehog) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java]: True ! By addressing one of Gabriel's comment, the function had to go anyway 🙂

**Review by jbsv — APPROVED** [review]

**Review by GabrielFleischer — APPROVED** [review]
LGTM

**Pull request merged**